GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -1081,5 +1075,40 @@ public static void TerminateTree(this Process process)
                 process.Kill();
             }
         }
+
+        public static string FormatCommitMessageFromTextBox(
+            string commitMessageText, bool usingCommitTemplate, bool ensureCommitMessageSecondLineEmpty)
+        {
+            if (commitMessageText == null)
+            {
+                return string.Empty;
+            }
+
+            var formattedCommitMessage = new StringBuilder();
+
+            var lineNumber = 1;
+            foreach (var line in commitMessageText.Split('\n'))
+            {
+                string nonNullLine = line ?? string.Empty;
```

Comment:
How can `line == null`?

Reply:
This is a preexisting method that I pulled from `GitUI/CommandsDialogs/FormCommit.cs`